sdcm/cluster_aws.py

```diff
@@ -421,15 +421,20 @@ def __init__(self, ec2_instance, ec2_service, credentials, parent_cluster,  # py
                           'key_file': credentials.key_file}
         self._spot_aws_termination_task = None
 
+        if len(self._instance.network_interfaces) == 2:
```

Comment:
Is it correct?
do we want to only wait the public ip for instance that has two network interfaces?

Reply:
It's incorrect, I wanted to reverse the condition